packages/configs/default/package.json

```diff
@@ -22,6 +22,7 @@
     "@parcel/transformer-babel": "^2.0.0-alpha.0",
     "@parcel/transformer-css": "^2.0.0-alpha.0",
     "@parcel/transformer-html": "^2.0.0-alpha.0",
+    "@parcel/transformer-posthtml": "^2.0.0",
```

Comment:
Can you make this use an alpha version?

Reply:
Is there a "fancy" way of doing that, or should I just update version, manually?

Reply:
I think just manually here and in the posthtml package.json